### PR TITLE
Updated Greek Translation

### PR DESCRIPTION
**Pull request by CedArctic**
Updated the Greek translation for #130

### CODE REVIEW

**Review by inxomnyaa — CHANGES_REQUESTED** [review]
Waiting for the missing translations :p

**Comment by CedArctic** [comment]
Will work on them and let you know asap

File: resources/lang/ell.ini [hunk 1]
```diff
@@ -72,30 +72,30 @@ flags.natural = "Φυσική"
...
+tool.debug.lore.1 = "Κάντε αριστερό κλικ στο block για να πάρετε πληροφορίες"
+tool.debug.lore.2 = "σαν το όνομα και το ποσό ζημιάς ενός block"
+tool.debug.lore.3 = "Χρησιμοποιείστε το //toggledebug να αλλάξετε τη λειτουργία του"
+tool.debug.disabled = "Το εργαλείο αποσφαλμάτωσης είναι απενεργοποιημένο. Χρησιμοποιείστε το //toggledebug για να το επανενεργοποιήσετε"
+tool.debug.setenabled = "Το εργαλείο αποσφαλμάτωσης είναι τώρα {%0}!"
 ; flood tool
-ui.flood.title = "Flood menu"
+ui.flood.title = "Μενού πλημμύρας"
 ui.flood.options.limit = "Μέγιστα blocks"
 ui.flood.options.blocks = "Blocks"
```
Comment (inxomnyaa) [inline resources/lang/ell.ini]: Thinking of the blocks, is it like a "specific" term in greece? Blocks are the breakable Voxels in the game Minecraft, i am unsure if minecraft has a specific word for them in greece. Maybe a fitting translation can be found here: https://mcpedl.com/translations-for-minecraft/?cookie_check=1

Comment (CedArctic) [reply resources/lang/ell.ini]: I've been thinking about it and probably the best localization is "τουβλάκια". I'm pushing a commit on this

**Review by inxomnyaa — APPROVED** [review]

**Pull request merged**